apis/Google.Cloud.Diagnostics.AspNetCore/docs/index.md

```diff
@@ -83,11 +79,11 @@ as described below
 ## Initializing Logging
 
 Logging can be initialized using `ConfigureServices` when
-constructing a `WebHostBuilder()`:
+constructing an `IWebHostBuilder`:
```

Comment:
Not sure that "constructing" works with the interface (here or below) "using an `IWebHostBuilder`?